pipelines/rnaseq.yaml

```diff
@@ -2,11 +2,16 @@
 resources:
   adapters: /data/groups/lab_bock/shared/resources/adapters/nextera_adapters.fa
   kallisto_index:
-    hg38_cdna: /data/groups/lab_bock/shared/resources/genomes/hg38_cdna/indexed_kallisto/hg38_cdna.index
-    hg19_cdna: /data/groups/lab_bock/shared/resources/genomes/hg19_cdna/indexed_kallisto/hg19_cdna.index
-    mm10_cdna: /data/groups/lab_bock/shared/resources/genomes/mm10_cdna/indexed_kallisto/mm10_cdna.index
-    mm9_cdna: /data/prod/ngs_resources/genomes/mm9_cdna/indexed_kallisto/mm9_cdna.index
-    ercc_cdna: /data/groups/lab_bock/shared/resources/genomes/ERCC92_cdna/indexed_kallisto/ERCC92_cdna.index
+    hg38: /data/groups/lab_bock/shared/resources/genomes/hg38_cdna/indexed_kallisto/hg38_cdna.idx
```

Comment:
for future reference, it's much better to route the reference genome annotations through the project config, so users of this pipeline don't have to fiddle with this script to point at a reference index on their own computer.  also if you use refgenie, pypiper can handle all this for you and you don't need to hardcode any of this stuff.

Reply:
See for an example, this kallisto pipeline (which was originally based on this one). https://github.com/databio/rnapipe

Reply:
Thanks, will look at it soon.